cmake/CTestConfig.cmake

```diff
@@ -2,7 +2,7 @@
 # CTest
 #
 
-set(PRECICE_TEST_TIMEOUT_LONG 120 CACHE STRING "The timeout in seconds for longer tests.")
+set(PRECICE_TEST_TIMEOUT_LONG 300 CACHE STRING "The timeout in seconds for longer tests.")
```

Comment:
@fsimonis I quickly checked the time required by the tests in #1414 and the maximum seems to be somewhere around 130s. Are the 300s too generous? I would like to avoid having to update this again when I add more tests.

Reply:
Try to keep the timeout as tight as practical.
I would jump to 120 to 180 seconds as a timeout.

Tight timeouts prevent hanging tests from using too much CPU time. We are using the official runners, so we are limited by how many can run in parallel.